src/app/components/Citations/formatCitation.jsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+
+function withCitationFields(WrappedComponent) {
```

Comment:
At a high level, this class seems to be taking props.edition and transforming it into something where you can get Title, etc.  

I think this is better suited to be a utility class like `EditionCard.jsx` where it just exposes a bunch of functions like (getAuthor), (getTitle), etc.  There might be overlap with `EditionCard`? Though I think it's not a bad thing to keep the two separate even if they happen to be returning the same results right now.